README.md

```diff
@@ -6,36 +6,31 @@ IPLD is not a single specification, it is a set of specifications.
-                +-------------+ |                             |
-                |             | | End-User Application Stacks |
-                | MFS in IPFS | |                             |
-                |             | +-----------------------------+
-                +-------------+ |                             |
-                |             | | Structured Data w/ indexes  |
-                |  unixfs v2  | |     VR, Geo, SQL, etc.      |                +----------+
-                |             | |                             |                |          |
-                +-------------+ +-----------------------------+                |  MFS in  |
-                |             | |                             |                |   IPFS   |
-                |    HAMT     | |   Sorted Index (sharded)    |                |          |
-                |             | |                             |                +----------+
-                +-------------+-+-----------------------------+                |          |
-                |                                             |                |  unixfs  |
-                |          Complex Data Structures            |                |    v1    |
-                |                                             |                |          |
-+-----------------------------------------------------------------------------------------+
-|               |                                             |                |          |
-|               |        dag-json         dag-cbor            |    ipld-git    |          |
-|               |                                             |                |          |
-|    Codecs     +---------------------------------------------+    ipld-btc    |  dag-pb  |
-|               |                                             |                |          |
-|               |               IPLD Data Model               |   ipld-zcash   |          |
-|               |                                             |                |          |
-+-------------------------------------------------------------+----------------+----------+
-                |                                                                         |
-                |                         Block (CID, Raw Data)                           |
-                |                                                                         |
-                +-------------------------------------------------------------------------+
+           ┌───────────┐ ┌────────────────────────────────────┐
+           │           │ │                                    │
+           │    MFS    │ │       End-User Applications        │
+           │           │ │                                    │
+           ├───────────┤ ├─────────────────┬──────────────────┤            ┌───────────┐
+           │           │ │ Structured Data │     Utility      │            │           │
```

Comment:
“Utility Collections” should actually just replace “Complex Data Structures.”

Reply:
I think I need to understand the geo etc. stuff better. I'm thinking the name "Specialized Data Structures" might be more suited and maybe "End-User Applications" should wrap around it so it's still also sitting on "Utility Collections".

So, I get GeoSpacial, but I don't know what "VR" is referring to in this context and I'm not sure "SQL" makes sense under that heading because it's an interfacing method rather than a collection type. @mikeal can you expand on the thoughts behind this category a bit?

Reply:
I agree with @rvagg version here. That's also my view on things. The old version was wrong anyway. A Geo index is *not* a sorted index.